### DISCUSSION TITLE
Update via web browser stopped working, update button does nothing

### DISCUSSION THREAD

**Discussion by i-do** [datenstrom/community#694] [Report a bug] [answered]
Interesting situation I have here. 
After Yellow 0.8.17 automatic updates stoped working.

I manually upload new version of Update plugin — now it's doesn't update from time to time.
Sometimes new plugins updates well, sometimes are not. So I can't find the reason for this.

Now I see the new version of Edit 0.8.53 but it doesn't updated after I press the Update button.

Log files doesn't show any errors.

Latest updates entries:
2021-08-17 17:16:37 info Update extension 'Edit 0.8.52'
2021-08-17 17:16:37 info Update extension 'Update 0.8.54'
2021-08-21 02:50:32 info Update extension 'Command 0.8.30'
2021-08-21 02:50:32 info Update extension 'Core 0.8.51'
2021-08-21 02:50:32 info Update extension 'Csv 0.8.14'
2021-08-21 02:50:32 info Update extension 'Update 0.8.55'
2021-08-21 02:50:32 info Update Datenstrom Yellow 0.8.18, PHP 7.1.33, Apache 2.4.10, Linux
2021-08-24 11:10:40 info Update extension 'Update 0.8.56'

Any idea how I can fix this?

**Comment by markseuffert** [comment, marked as answer]
Thanks for the detailed bug report. The problem was introduced a few weeks ago when I changed the update extension, but forgot to make corresponding changes in the edit extension. As a result updates were no longer possible via the web browser, because it didn't send the required "update all" command to the backend. In other words, there was a miscommunication between two parts of the system. Must have slipped trough testing, I am really sorry about that.

[Manually download Edit 0.8.54](https://github.com/datenstrom/yellow-extensions/raw/3142fec3485f2218ac022495ae4bef056ac24587/zip/edit.zip) and copy zip file into your `system/extensions` folder. Right click if you use Safari. 

This should fix it.

**Reply by i-do** [reply]
Yes, now it works.

So it will be no more updates via browser, only at command line method? 
Or because of Edit extension was not changed?

**Reply by markseuffert** [reply]
You can update your website via web browser and command line. Datenstrom Yellow supports both.

**Comment by dajare** [comment]
I can't update from the website link (`admin > Settings > Updates`), and wonder whether my issue is related to this.

I applied "edit.zip" to my `system/extensions` folder, and it is `VERSION = "0.8.54"`.

When I click on the "Updates" link, I see:

> Datenstrom Yellow 0.8.18
> 
> The following updates are available:
> 
> Command 0.8.30
> Update 0.8.56

But when I click the "Update" button, I get: 

> **Server error**
> 
> Something went wrong. Can't find extension 'all'!

Are there other core files I should update to restore this functionality? Thanks!

**Reply by schulle4u** [reply]
In case of this error message, manually download [update v0.8.56](https://raw.githubusercontent.com/datenstrom/yellow-extensions/4b0db7b3d05b003b0d4ee8cdf7ed348feee628e9/zip/update.zip) and upload the zip file into your `system/extensions` directory. 
**Note**: I'm not fully sure what could have lead into this specific issue, but in any other setups the normal update procedure should be used.

**Reply by dajare** [reply]
👍 You've sorted me out again! 😄 Thanks. I wondered if that was it, but was a little nervous about guessing. All good now!
